.circleci/config.yml

```diff
@@ -22,14 +22,14 @@ commands:
     downstream:
         steps:
             - checkout
-            # - restore_cache:
-            #       keys:
-            #           - v4c-dependencies-{{ arch }}-{{ checksum "yarn.lock" }}
-            # - restore_cache:
-            #       keys:
-            #           - v4b-<< pipeline.parameters.wireit_cache_name >>-{{ arch }}-{{ checksum "package.json" }}-
+            - restore_cache:
```

Comment:
i remeber this getting commented out, but don't remember why. can you share whats going on here so I can better understand? I know its calling the cache but more understanding of why it will work now would be awesome :) 

I want this efficiency :D